packages/ui-scripts/lib/build/buildThemes/setupThemes.js

```diff
@@ -152,12 +141,22 @@ const setupThemes = async (targetPath, input) => {
           importSemantics = `import type { Semantics } from "../${theme}/semantics"`
         }
         let importBoxShadow = ''
-        if (componentTypes.includes(boxShadowType)) {
-          importBoxShadow = `import type { BoxShadow } from '../commonTypes'\n`
+        if (componentTypes.includes('TokenBoxshadowValueInst')) {
+          importBoxShadow = `import type { TokenBoxshadowValueInst } from '../commonTypes'`
+        }
+        let importBorder = ''
+        if (componentTypes.includes('TokenBorderValue')) {
+          importBorder = `import type { TokenBorderValue } from '@tokens-studio/types'`
+        }
+        let importTypography = ''
+        if (componentTypes.includes('TokenTypographyValueInst')) {
+          importTypography = `import type { TokenTypographyValueInst } from '../commonTypes'`
```

Comment:
This is a bit ugly, but simple. If we want we can make this nicer and dynamic, IMO its not needed because I dont expect Token Studio to change its types often